web/styles/popovers.css

```diff
@@ -1074,6 +1035,85 @@ ul {
     }
 }
 
+.popover-menu-user-header {
+    display: flex;
+    flex-flow: row nowrap;
+    gap: 7px;
+    text-align: left;
+    padding: 4px;
+
+    .popover-menu-user-avatar-container {
+        position: relative;
+        flex-shrink: 0;
+        width: 64px;
+        height: 64px;
+    }
+
+    .popover-menu-user-avatar {
+        width: 64px;
+        height: 64px;
+        border-radius: 4px;
+        background-size: cover;
+        background-position: center;
+    }
+
+    .popover-menu-user-presence {
+        position: absolute;
+        width: 8px;
+        height: 8px;
+        right: -1px;
+        bottom: -1px;
+        border: solid 1px var(--color-background);
+        border-radius: 50%;
+    }
+
+    .popover-menu-user-full-name {
+        font-size: 18px;
+        font-weight: 600;
+        line-height: 20px;
+        color: var(--color-text-full-name);
+        word-break: break-word;
+    }
+
+    .popover-menu-user-type {
+        font-size: 14px;
+        font-weight: 400;
+        line-height: 16px;
+        color: var(--color-text-item);
+    }
+
+    .user_circle_empty {
+        background-color: var(--color-background-popover-menu);
+        border-color: hsl(0deg 0% 50%);
+    }
+}
+
+.user-email-field {
+    transition: all 0.4s ease;
+
+    & .user_popover_email {
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        overflow: hidden;
+    }
+
+    & i {
+        cursor: pointer;
+
+        &.hide_copy_icon {
+            display: none;
+        }
+
+        &:hover {
+            color: hsl(170deg 48% 54%);
+        }
+    }
+
+    &.email_copied {
+        color: hsl(170deg 48% 54%) !important;
```

Comment:
I have kept the color change on the email being copied intact, but this
color is standout from the rest of the UI, so we should discuss about it at CZO.

Reply:
Ooh, yeah, I don't think we want this. I think we should just use a "Copied!" tooltip for consistency with the copying confirmations across the UI.

Reply:
Let's check with @terpimost about the icon hover color as well (we can discuss in #design). I haven't checked the values, but it looks the same as here?

![Screenshot 2024-07-02 at 13 00 13](https://github.com/zulip/zulip/assets/2090066/6d641428-8019-45c3-8529-715c718905cb)